core/egov-localization/src/main/java/org/egov/domain/service/MessageService.java

```diff
@@ -104,13 +103,22 @@ public void bustCache() {
         return messages;
     }
     
-    public boolean createMessage(CreateMessagesRequest createMessagesRequest){
-    	messageRepository.deleteMessages(createMessagesRequest);
-   		boolean createStatus = messageRepository.createMessage(createMessagesRequest);
+    public boolean createMessage(String locale, String tenantId, List<Message> messageList){
```

Comment:
@darshannagesh Tenant and locale is present within each message. Why do we need it to be passed as separate parameters? The whole intent of having a message domain model is that it represents a combination of locale, module, tenant, code and message.